tl/cli/check-candidates.py

```diff
@@ -0,0 +1,76 @@
+import traceback
+import argparse
+import sys
+import tl.exceptions
+from tl.exceptions import UnsupportTypeError
+from tl.utility.logging import Logger
+
+
+def parser():
+    return {
+        'help': 'Checks if for each row the ground truth was retrieved'
+                ' and outputs those rows for which the ground truth was never'
+                ' retrieved'
+    }
+
+
+def add_arguments(parser):
+    # input file
+    parser.add_argument('input_file', nargs='?', type=argparse.FileType('r'),
+                        default=sys.stdin)
+    parser.add_argument('--gt-file', type=str, dest='gt_file', required=True,
+                        help="ground truth file to be compared against")
+
+
+def run(**kwargs):
+    try:
+        import pandas as pd
```

Comment:
first of all, the logic should be in a separate file since we need to use this as an API as well.

Second, I was thinking about a simpler way of doing this. The command `ground-truth-labeler` already marks the GT for each cell, adding a column `evaluation_label` ( 1, 0 , -1). For this command, you just have to check if for a set of candidates for a cell, there exists a candidate which has `evaluation_label == 1`

Keep this simple, please change